server/routes/punishments/activateSuspendedPunishments/activateSuspendedPunishmentsPage.ts

```diff
@@ -22,12 +23,22 @@ export default class ActivateSuspendedPunishmentsPage {
       susPun => susPun.reportNumber !== adjudicationNumber
     )
 
+    const sessionPunishments = await this.punishmentsService.getAllSessionPunishments(req, adjudicationNumber)
+
+    let suspendedPunishmentsToActivate = suspendedPunishmentsFromOtherReports
+    if (sessionPunishments != null) {
+      const idsToFilter = (<PunishmentDataWithSchedule[]>sessionPunishments).map(m => m.id)
+      suspendedPunishmentsToActivate = suspendedPunishmentsFromOtherReports.filter(
+        f => !idsToFilter.includes(f.punishment.id)
+      )
+    }
+
```

Comment:
Any chance we could give the variables 'm' and 'f' more meaningful names? I find it really hard to follow when names are only one letter!